### DISCUSSION TITLE
Data hygiene question: Switchboard

### DISCUSSION THREAD

**Discussion by ekayen** [openai/whisper#680] [Q&A] [answered]
I am hoping to use Whisper for a research project, and I need to make sure that it hasn't been trained on the data I'll be using for evaluation. I'm using Switchboard NXT (because of the rich annotation set available), and will be evaluating on the conventional dev and test sections of the corpus. I couldn't find info in the paper about whether existing corpora like Switchboard were used, though I may have missed it. Is Switchboard included in Whisper's training data? If so, was all of it included, or was some portion of dev/test data omitted?

**Comment by jongwook** [comment, marked as answer]
Hi, we did not use Switchboard as part of our training mix. However, due to the noisy nature of the large training dataset, there might be a small number of samples of the audio from the dataset scattered into the mix.

**Reply by ekayen** [reply]
Thanks for the reply! That's very helpful.
